tests/functional/aws-node-sdk/test/bucket/listingCornerCases.js

```diff
@@ -475,4 +475,56 @@ describe('Listing corner cases tests', () => {
             }
         });
     });
+
+    it('should not list DeleteMarkers for version suspended buckets', done => {
+        const obj = { name: 'testDeleteMarker.txt', value: 'foo' };
+        let objectCount = 0;
+        const Bucket = `versioning-bucket-test-delete-marker-${Date.now()}`;
```

Comment:
There's already a global called `Bucket`, it could be better to rename this new bucket to avoid confusion.